### DISCUSSION TITLE
Is TileDesk is completely opensource?

### DISCUSSION THREAD

**Discussion by xcellenceit** [Tiledesk/tiledesk#29] [General]
Wanted to know if TileDesk is completely opensource? Or there are components / modules that are not opensource and only offered on cloud version?

Anywhere we can see the comparison for both?

**Comment by MicheleTiledesk** [comment]
Tiledesk is open-source at its core, but the cloud-hosted SaaS version includes some additional advanced features not available in the self-hosted open-source version. 

Specifically, the following modules are not included in the open core:
- Integration with WhatsApp Business and Facebook Messenger Channels
- Integration with Email Channel
- Integration with SMS Channel 
- Ask the Knowledge Base
- Support for VoiceXML or similar languages
- Single-Sign-On (SSO)
- Configuration of the Module for Mass Sending of Outbound WhatsApp and SMS Messages
- Advanced Queue Management (Smart Assignment)

**Comment by CKaranD** [comment]
Is there any conversation limit for the self-hosted open-source version?
